en/05_Contributing/00_Issues_and_Bugs.md

```diff
@@ -1,96 +1,77 @@
+write it for you. Any "wishlist" type issues without code attached can be
 expected to be closed as soon as they're reviewed.
 [/warning]
 
-In order to gain interest and feedback in your feature, we encourage you to 
+In order to gain interest and feedback in your feature, we encourage you to
 present it to the community through the [community channels](https://www.silverstripe.org/community).
 
 ## Reporting Security Issues
 
+[warning]
 If you think a bug may have security implications, do not create a GitHub issue for it. This may lead to a zero-day vulnerability.
+[/warning]
 
-Report potential security issues to [security@silverstripe.org](mailto:security@silverstripe.org). 
-See our "[Release Process](/contributing/release_process/#security-releases)" documentation for more info, and 
-read our guide on [how to write secure code](/developer_guides/security/secure_coding/).
+Report potential security issues to [security@silverstripe.org](mailto:security@silverstripe.org). Emails sent to that address are
+forwarded to a private mailing list and kick off a specific security process.
 
-Silverstripe CMS does not operate a *bug bounty* program.
+If you have any doubts or are unsure whether the bug you've found has security implications or not, please err on the side of caution
+and email us about it.
 
-Review our [Managing Security Guidelines](managing_security_issues) guidelines to understand what happens once a vulnerability is reported.
+Review our [Managing Security Issues](managing_security_issues) process to understand what happens once a vulnerability is reported.
 
 Silverstripe CMS aims to ship security patches at pre-defined intervals when those issues are not actively exploited in the wild.
-
 Review the [Security patch windows](../Project_Governance/Minor_release_policy#security-patch-windows) section of our minor release policy for more details.
 
-## Sharing your Opinion
-
-* [forum.silverstripe.org](http://forum.silverstripe.org): Forums on silverstripe.org
-* [All issues across modules](https://www.silverstripe.org/community/contributing-to-silverstripe/github-all-core-issues)
-
-## Identifying issues and pull request relevant to your own project {#identifying-issues}
-
-Our [issue browser](https://silverstripe-github-issues.now.sh/) can be helpful to identify known issues and pending pull requests in
-[supported modules](https://www.silverstripe.org/software/addons/silverstripe-commercially-supported-module-list/).
-But you're usually only running some of these modules, and others
-from the [wider module ecosystem](https://addons.silverstripe.org).
-
-In order to only show issues and pull requests relevant to your project,
-we've written a little [composer utility](https://github.com/silverstripe/silverstripe-github-issue-search-composer-util)
-which inspects your own `composer.lock` file dependencies,
-and searches across all Silverstripe CMS modules in there.
```

Comment:
This isn't a "how to find bugs that might affect my project" page, and the tool noted here isn't maintained.